sway-lsp/src/traverse/typed_tree.rs

```diff
@@ -519,16 +519,10 @@ impl<'a> TypedTree<'a> {
                     self.collect_call_path_prefixes(&call_path.prefixes);
                 }
 
-                let span = if let Some(call_path) = call_path {
```

Comment:
Is there any reason to remove this check? We want to make sure if the call path looks something like `foo::bar::const` that the span only contains the `const` token and not the `foo::bar` span info.

Reply:
I see your point. What I did is update the span of VariableExpression to always point to the name of the const in the expression even if a call path is provided, so the check is no longer necessary here. But I'm now wondering it this is the right approach. The span of the suffix of the call paths may point to the const declaration after my changes. I think I can do better here though.

Reply:
Merging for now and will deal with the span thing tonight or tomorrow.

Reply:
Ok great, happy for this to be addressed in a follow up PR so @kayagokalp can be unblocked.